390/Phuong_Chau.Rmd

```diff
@@ -0,0 +1,44 @@
+---
+title: "mp2-visualization"
+author: "Phuong Chau"
+date: "9/17/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Load libraries
+```{r, echo = FALSE, warning=FALSE}
+library(readr)
+library(tidyverse)
+library(dplyr)
+```
+
+```{r, echo = FALSE, warning=FALSE}
+# Load data and remove NA values
+stem1 <- read_csv("C:/Fall2020_Lastsemester/EcologicalFocasting/SCBI-ForestGEO-Data/tree_main_census/data/census-csv-files/scbi.stem1.csv")
```

Comment:
Hey Candy, here is a code review!

Unfortunately, this code doesn't work on my computer b/c of the filepath. Check out this comment I made on Lauren's PR that will set up code that will work on anyone's computer https://github.com/390-2020-09-Fall/SCBI-ForestGEO-Data/pull/2#discussion_r491179003

Reply:
Could you fix this and push to this branch?

Reply:
Once you fix this and the .Rmd file knits, I'll merge into `master`

Reply:
Hi Prof. Kim! I have changed the read_csv() and committed again but I don't see the "Compare and pull request" anymore. What should I do next after I committed?

Reply:
Great question! When you make commits to a still unmerged pull request, the pull request auto-updates. In other words, you don't need to make a new one.

But if you make changes to a branch that's already been merged, then you will need to create a new PR.